libavutil/version.h

```diff
@@ -80,7 +80,7 @@
 
 #define LIBAVUTIL_VERSION_MAJOR  57
 #define LIBAVUTIL_VERSION_MINOR  24
-#define LIBAVUTIL_VERSION_MICRO 101
+#define LIBAVUTIL_VERSION_MICRO 102
 
```

Comment:
Please bump the minor version instead of the micro version, see the requirement in https://ffmpeg.org/developer.html#Code